### PR TITLE
Resume verification button hidden on revoked projects

### PR DESCRIPTION
**Pull request by mateodaza**

### CODE REVIEW

File: src/components/views/project/projectDonateCard/ProjectDonateCard.tsx [hunk 1]
```diff
@@ -283,14 +282,16 @@ const ProjectDonateCard: FC<IProjectDonateCard> = ({
...
-						{isVerDraft && (
-							<ExternalLink href={slugToVerification(slug)}>
-								<FullOutlineButton
-									buttonType='primary'
-									label='RESUME VERIFICATION'
-								/>
-							</ExternalLink>
-						)}
+						{verificationStatus !==
+							EProjectVerificationStatus.REVOKED &&
```
Comment (RamRamez) [inline src/components/views/project/projectDonateCard/ProjectDonateCard.tsx:286]: @CarlosQ96 said `verificationStatus` will be changed with `verified` status at the same time with cronjob. Means if project verification is revoked, `verified === false` also. So I think you don't need `verificationStatus` for this task anymore.
For showing `VERIFY YOUR PROJECT` button, I think the `verified` condition is enough and you can remove `verificationStatus` condition.
And for showing `RESUME VERIFICATION` button, I think replacing `verificationStatus !== EProjectVerificationStatus.REVOKED` condition by `verified` will work.

**Review by RamRamez — CHANGES_REQUESTED** [review]
Thanks @mateodaza. Your solution is working well. I have a suggestion please take a look.

File: src/components/views/project/projectDonateCard/ProjectDonateCard.tsx (hunk 1 above)
Comment (mateodaza) [reply src/components/views/project/projectDonateCard/ProjectDonateCard.tsx:286]: There are two cases 
- A new project that has never been verified, the property is shown as false but we want to display the button
- A previously verified project which was revoked. We don't want them to see the verification button

If I don't check both values the first case will never see the option

Comment (RamRamez) [reply src/components/views/project/projectDonateCard/ProjectDonateCard.tsx:286]: Is it necessary to check `verificationStatus` for showing `RESUME VERIFICATION` button? I mean the project is not verified yet and verificationStatus can't be revoked.

**Review by RamRamez — APPROVED** [review]

**Pull request merged**